spec/models/section_spec.rb

```diff
@@ -110,6 +110,21 @@ def key_classes_for(hash)
     allow(SlugGenerator).to receive(:new).with(prefix: manual_slug).and_return(slug_generator)
   end
 
+  describe '#update_slug!' do
+    it 'updates the slug of the section' do
+      allow(SlugGenerator).to receive(:new).and_call_original
```

Comment:
I don't understand why you need this. If you really do need it, can you explain why in the commit note?